docs/developing_recipes_locally.md

```diff
@@ -0,0 +1,201 @@
+# Developing Recipes Locally
+
+One of the early steps for contributing get getting familiar with the Conan client. There's also a few steps you need to take to get your environment setup.
+
+This file is intended to provide all the commands you need to run in order to be an expert ConanCenter contributor.
+
+<!-- toc -->
+## Contents
+
+  * [Clone your fork](#clone-your-fork)
+  * [Setup your environment](#setup-your-environment)
+    * [Installing the ConanCenter Hooks](#installing-the-conancenter-hooks)
+      * [Updating conan hooks on your machine](#updating-conan-hooks-on-your-machine)
+  * [Basic Commands](#basic-commands)
+    * [Try it yourself](#try-it-yourself)
+  * [Debugging Failed Builds](#debugging-failed-builds)
+  * [Running the Python Linters](#running-the-python-linters)
+  * [Testing the different `test__package`](#testing-the-different-test__package)
+  * [Testing more environments](#testing-more-environments)
+  * [Using Conan 2.0](#using-conan-20)<!-- endToc -->
+
+## Clone your fork
+
+1. Follow the GitHub UI to [fork this repository](https://github.com/prince-chrismc/conan-center-index/fork)
+2. [Clone your fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+## Setup your environment
+
+1. Install a C++ development toolchain - ConanCenter's [build images](#testing-more-environments) are available
+2. [Install the Conan client](https://docs.conan.io/en/latest/installation.html) - make sure to keep it up to date!
```

Comment:
I feel like a 

> **Note**: Conan always recommends working within a `venv` to ensure python dependencies do not collide 

might be useful... especially for adding a separate 2.0 setup below